app/src/test/java/org/mozilla/fenix/tabstray/browser/AbstractBrowserTabViewHolderTest.kt

```diff
@@ -41,7 +45,6 @@ class AbstractBrowserTabViewHolderTest {
             store,
             null,
             browserStore,
-            mockk(relaxed = true),
             interactor
         )
 
```

Comment:
Is it possible to verify that after `performClick` we recorded the right telemetry?